package.json

```diff
@@ -19,8 +19,8 @@
     "format:php": "composer format",
     "format:js": "npm run lint:js -- --fix",
     "format:css": "npm run lint:css -- --fix",
-    "test": "npm-run-all test:*",
-    "test:php": "npm run composer test",
+    "test": "npm run composer test",
+    "test:watch": "npm run composer test:watch -- -- --timeout=0",
```

Comment:
While resolving the merge conflict, I was able to move the timeout logic here where I assume it was needed. Please correct me if I'm wrong.

Reply:
Ah that's a great idea, I'll test that out 👍🏻